Marlin/Marlin.h

```diff
@@ -110,6 +110,8 @@ FORCE_INLINE void serialprintPGM(const char *str) {
 
 void get_command();
 void process_commands();
+uint16_t gcode_M238(uint8_t power = 0);
+void gcode_M241(long num_milliseconds = NULL);
```

Comment:
The gcode functions are usually inlined and defined in Marlin_main.cpp. Any reason you want to put the defs here?

Also, you should probably put some `#ifdef` stuff around them if you are using the distance sensor or anything else that isn't in use by default.